fe/model.py

```diff
@@ -65,18 +65,6 @@ def predict(self, ff_params: list, mol_a: Chem.Mol, mol_b: Chem.Mol, core: np.nd
         core: np.ndarray
             N x 2 list of ints corresponding to the atom mapping of the core.
 
-        callback: function
-            accepts a list of SimulationResults and a string, and doesn't return anything. may save to disk
-            TODO: is there a way to save intermediate results to disk that doesn't require passing a callable a few layers deep here...
-            TODO: aux return following https://github.com/proteneer/timemachine/pull/360#discussion_r584733991
-                * Update RBFEModel.predict() and estimator.deltaG() signature to:
-                    * accept an options argument that determines what gets put into the aux return
-                    * forward aux return
-                * options object to determine return types:
-                    * three boolean flags (return xs, return du_dls, return du_dps)
```

Comment:
Resolved in favor of returning all 3 -- correct?

Reply:
yeah lets figure out how to partially specify what parts of the SimulationResult should be filled out later on.